CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Built-on-column-plugin
+
+This plugin Adds a column on the jenkins dashboard that shows the actual node the last build was run on.
+
+# Why should you contribute
+
+You can contribute in many ways, and whatever you choose we're grateful! Source code contribution is the obvious one but we also need your feedback and if you don't really want to participate in the implementation directly you may still have great ideas about features we need (or should get rid of).
```

Comment:
I prefer sentence per line because it makes code review easier.  Asking for suggestions of feature removals is much less likely to have positive responses, since feature removal is a breaking change and we avoid breaking changes.
```suggestion
You can contribute in many ways, and whatever you choose we're grateful
Source code contribution is the obvious one but we also need your feedback and if you don't really want to participate in the implementation directly you may still have great ideas about features we need.
```